python/cugraph/cugraph/gnn/pyg_extensions/data/cugraph_store.py

```diff
@@ -285,18 +288,26 @@ def is_multi_gpu(self):
         """
         return self.__graph.is_multi_gpu()
 
+    @cached_property
+    def is_remote(self):
+        return self.__graph.is_remote()
+
+    @cached_property
+    def _compute_required(self):
+        return self.is_multi_gpu and not self.is_remote
```

Comment:
**Nitpick:**
I really like the idea of having something like a `compute_required` but i am not sure about the name. Maybe something like a `_is_delayed_object` .  Feel free to ignore.

Reply:
changed to `_is_delayed`